### PR TITLE
GH Actions Testing Workflow

### PR DESCRIPTION
**Pull request by taylorfturner**
Miscellaneous fixes to get unit testing / pre-commit checks running on PR creation:
- typo
- isort version bump
- less restrictive `python_version` in `Pipfile` (`3.9` --> `3`)

### CODE REVIEW

File: clients/python_client/Pipfile [hunk 1]
```diff
@@ -15,4 +15,4 @@ tox = "*"
 pre-commit = "*"
 
 [requires]
-python_version = "3.9"
+python_version = "3"
```
Comment (taylorfturner) [inline clients/python_client/Pipfile:18]: https://github.com/pypa/pipenv/issues/1050 `pipenv` has declined to support `PEP 508 specifiers`

File: connectors/django/.pre-commit-config.yaml [hunk 2]
```diff
@@ -32,7 +32,7 @@ repos:
         exclude: (^.*/tests|^.*/__init__.py|connectors/django/fma_django/migrations)
 
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort for django
```
Comment (taylorfturner) [inline connectors/django/.pre-commit-config.yaml:38]: required bump due to a bug we have seen in the past

File: fma-core/fma_core/conf.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ def __init__(self):
         agg_settings = getattr(mod, "AGGREGATOR_SETTINGS", None)
         if not agg_settings or not isinstance(agg_settings, dict):
             raise ValueError(
-                "`FMA_SETTINGS_MODULE` is imporpery configured and must "
+                "`FMA_SETTINGS_MODULE` is improperly configured and must "
```
Comment (taylorfturner) [inline fma-core/fma_core/conf.py:19]: typo

**Review by ksneab7 — APPROVED** [review]

File: aggregator/Pipfile [hunk 4]
```diff
@@ -21,4 +21,4 @@ fma-django = {path = "./artifacts/fma_django-0.0.1-py3-none-any.whl"}
 fma-core = {path = "./artifacts/fma_core-0.0.1-py3-none-any.whl"}
 
 [requires]
-python_version = "3.9"
+python_version = "3"
```
Comment (JGSweets) [inline aggregator/Pipfile:24]: Can we specify 3.8+?

File: .github/workflows/test-python-package.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Test fma-core on python version ${{matrix.python-version}}
+      run: |
+        cd fma-core
+        python -m pip install pipenv
```
Comment (JGSweets) [inline .github/workflows/test-python-package.yml:25]: can we move this to a presetup before the tests?

File: aggregator/Pipfile (hunk 4 above)
Comment (taylorfturner) [reply aggregator/Pipfile]: no dice ```ValueError: invalid literal for int() with base 10: '3.8+'```

File: .github/workflows/test-python-package.yml (hunk 5 above)
Comment (taylorfturner) [reply .github/workflows/test-python-package.yml:25]: presetup meaning its own `- name:` section for setup?

Comment (taylorfturner) [reply .github/workflows/test-python-package.yml]: done

**Review by JGSweets — APPROVED** [review]

**Review by ksneab7 — APPROVED** [review]

**Pull request merged**